src/components/views/rooms/E2EIcon.tsx

```diff
@@ -78,9 +78,9 @@ const E2EIcon: React.FC<IProps> = ({
 
     let e2eTitle: string | undefined;
     if (isUser) {
-        e2eTitle = crossSigningUserTitles[status];
+        e2eTitle = crossSigningUserTitles[status!];
```

Comment:
If assuming, that the status is always set,
why changing the prop interface to make it optional?

Reply:
`{}[undefined] === undefined` so it was just a trick to skip needing another if case for if `status === undefined`